sunbird-cassandra-migration/cassandra-migration/src/main/resources/db/migration/cassandra/V1.79_cassandra.cql

```diff
@@ -1,5 +1,5 @@
-DROP TABLE sunbird_courses.assessment_aggregator;
-DROP TYPE sunbird_courses.question;
+DROP TABLE IF EXISTS sunbird_courses.assessment_aggregator;
```

Comment:
Is this file already run on dev then it won't again run , is this ok for you.

Reply:
yes we already tested this in dev